NOnion/Utility/EmbeddedResourceUtility.fs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NOnion.Utility
+
+open System
+open System.IO
+open System.Reflection
+
+module EmbeddedResourceUtility =
+
+    // Code from https://github.com/nblockchain/geewallet/blob/428cb77d21dba20fc38c7ea032003c5861aac950/src/GWallet.Backend/Config.fs#L156
+    let ExtractEmbeddedResourceFileContents(resourceName: string) : string =
```

Comment:
@aarani I wonder if we should put this in Fsdk too

Reply:
> > Directory: fix broken directory-signature parsing
> 
> Why was it broken?

Added to the commit msg

Reply:
Good idea though